test/src/org/labkey/test/components/targetedms/QCPlotsWebPart.java

```diff
@@ -169,6 +169,17 @@ private void toggleQCPlotTypes(Set<QCPlotType> plotTypes)
         }
     }
 
+    public String getTrailingLast()
+    {
+        return getWrapper().getFormElement(elementCache().trailingLast);
+    }
+
+    public void setTrailingLast(String value)
+    {
+        getWrapper().shortWait().until(ExpectedConditions.elementToBeClickable(elementCache().trailingLast));
+        getWrapper().setFormElement(elementCache().trailingLast, value);
```

Comment:
Wait for plots to update.
```suggestion
        doAndWaitForUpdate(() -> getWrapper().setFormElement(elementCache().trailingLast, value));
```

Reply:
done